flag.go

```diff
@@ -595,3 +596,27 @@ func withEnvHint(envVar, str string) string {
 	}
 	return str + envText
 }
+
+func stringifyFlag(f Flag) string {
+	fv := reflect.ValueOf(f)
+	placeholder, usage := unquoteUsage(fv.FieldByName("Usage").String())
```

Comment:
Could we make `FlagStringer` private? That would assuage my fears a little in that at least we keep the reflection internal to the library.